bigbluebutton-html5/package.json

```diff
@@ -67,7 +67,7 @@
     "reconnecting-websocket": "~v4.1.10",
     "redis": "~2.8.0",
     "string-hash": "~1.1.3",
-    "tippy.js": "^3.1.3",
+    "tippy.js": "^3.4.1",
```

Comment:
In future pls commit package-lock.json when making changes to package.json :wink: 
I pushed the missing changes in https://github.com/bigbluebutton/bigbluebutton/pull/6775/files#diff-f6771db7ba8f4b56eca53f46f6bcf119